clang/include/clang/StaticAnalyzer/Core/PathSensitive/SymbolManager.h

```diff
@@ -100,41 +100,7 @@ class SymbolConjured : public SymbolData {
   ConstCFGElementRef getCFGElementRef() const { return Elem; }
 
   // It might return null.
-  const Stmt *getStmt() const {
```

Comment:
moved this out of the header file because I feel like this is getting too big to be included in the header, let me know if you agree

Reply:
It must have been painful to iterate on this code :D
I agree, it should be in a cpp file.